examples/fungible-token/rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly
```

Comment:
I think you need to specify `nearcore` specific chain: `nightly-2020-03-19`

But instead we should use it only for the test folder. E.g. using `cargo +nightly-2020-03-19 ...`

Reply:
So `cargo +nightly-2020-03-19 test` ? or add `nightly-2020-03-19` to a file in the test folder?

Reply:
I added `+nightly-2020-03-19` to the `tests_all.sh` file and they worked.  Should I push that?

Reply:
There is already `rust_toolchain` in the root.

Reply:
Yes the issue was that the test won't build without nightly, so if it's ok for all the tests to be tested with nightly I can push or I just need to have a special case for this directory in `test_all.sh`.

Reply:
Oh, actually I was thinking that rust_toolchain in the root is `nightly-2020-03-19` (like in https://github.com/near/initial-contracts/blob/master/rust-toolchain). Looks like it's not a case. 

Having a local rust_toolchain with `nightly-2020-03-19` in the examples/fungible-token directory should be ok.